### PR TITLE
Update Magento_Captcha ReadMe

### PR DESCRIPTION
**Pull request by korovitskyi**
### Description (*)

Update README.md file for Magento_Captcha module.

### Fixed Issues (if relevant)

1. magento/devdocs magento/devdocs#4451: README content for modules - work in M2 Code

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Captcha/README.md [hunk 1]
```diff
@@ -1 +1,52 @@
...
+
+## Extensibility
+
+Extension developers can interact with the Magento_Captcha module. For more information about the Magento 2 extension mechanism, see [Magento plugins](https://devdocs.magento.com/guides/v2.4/extension-dev-guide/plugins.html).
+
+[The Magento 2 dependency injection mechanism](https://devdocs.magento.com/guides/v2.4/extension-dev-guide/depend-inj.html) enables you to override the functionality of the Magento_Captcha module.
+
+### Layouts
+
+This module introduces the following layouts and layout handles in the directories:
```
Comment (orlangur) [inline app/code/Magento/Captcha/README.md:27]: Nice, looks really useful :-)

**Review by orlangur — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @orlangur, thank you for the review.
[ENGCOM-8556](https://jira.corp.magento.com/browse/ENGCOM-8556) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
Hi @korovitskyi
Thank you for your contribution. This pull request has been combined (merged) with https://github.com/magento/magento2/pull/31304 to optimize pull requests queue on magento/magento2 repository
Closing this pull request.

**Comment by magento-engcom-team** [comment]
Hi @korovitskyi
Thank you for your contribution. This pull request has been combined (merged) with https://github.com/magento/magento2/pull/31304 to optimize pull requests queue on magento/magento2 repository
Closing this pull request.

**Pull request closed without merging**